third_party/BUILD

```diff
@@ -388,6 +388,16 @@ java_import(
     ],
 )
 
+# For bootstrapping JavaBuilder
```

Comment:
Remove this comment, it's not needed (and this will probably be used for other things).